kotlin-jvm/versions.properties

```diff
@@ -74,3 +74,5 @@ version.retrofit2=2.9.0
 version.sqldelight=1.4.3
 
 version.com.github.ajalt..mordant=1.2.1
+
+version.kotlinx.datetime=0.1.0
```

Comment:
Can you run `$ ./gradlew refreshVersions` ? 
This will re-order correctly the keys in `versions.properties`

Reply:
Fixed